src/libtorch.cc

```diff
@@ -424,6 +424,14 @@ ModelState::ParseParameters()
   return nullptr;
 }
 
+// The naming convention followed for inputs/outputs in the model configuration.
+// Outputs don't support ARGUEMENT.
+enum class NamingConvention {
```

Comment:
So we are sticking with NamingConvention instead of IOTensorOrder?

Reply:
Yes, in my opinion it is earlier to pair this with the Naming Convention detailed in the model configuration.